### ISSUE TITLE
Emulator proxy 

### ISSUE THREAD

**Issue by vaclavbenes** [google/android-emulator-container-scripts#103]
Hi is it possible set emulator proxy configuration? Like
`emulator -avd myavd -http-proxy http://168.192.1.2:3300`

**Comment by pokowaka** [comment]
Yes this is possible. Note that you cannot pass in the `-avd myavd` flag. The docker container uses a preconfigured Pixel2 avd.

You can pass in additional parameters as follows:

1. Create the container with the extra flag: 
   ```sh
   $ emu-docker create emuzip imgzip --extra "-http-proxy http://168.192.1.2:3300"
   ```

2. Or launch the container with additional parameters:
   ```sh
   $ ./run.sh <container_id> -http-proxy http://168.192.1.2:3300
   ```

**Comment by serpavlov** [comment]
Hi!
I'm using [docker-container-proxy](https://www.npmjs.com/package/docker-container-proxy), and it's working good with emulator

**Comment by kavabunga6** [comment]
Hi!
Proxy not working if i use adb shell settings put global http_proxy ip:port
We need set proxy without reboot or recreate image

**Comment by pokowaka** [comment]
You are likely running into docker networking issues, see: [Issue 140](https://github.com/google/android-emulator-container-scripts/issues/140)

**Comment by jpcottin** [comment]
You can pass arbitrary emulator arguments through `--extra` at `emu-docker create` time:

    emu-docker create <emu.zip> <sysimg.zip> --extra "-http-proxy" "http://168.192.1.2:3300"

Or, if you're using one of the prebuilt registry images, set the proxy via `EMULATOR_PARAMS` at `docker run` time (the launcher script appends `$EMULATOR_PARAMS` to the emulator command line):

    docker run -e EMULATOR_PARAMS='-http-proxy http://168.192.1.2:3300' ...

Closing as a configuration question.
